plugins/destination/postgresql/client/list_tables.go

```diff
@@ -88,7 +88,10 @@ func (c *Client) listTables(ctx context.Context) (schema.Tables, error) {
 			})
 		}
 		table := tables[len(tables)-1]
-		if pkName != "" {
+		// Note: constraints always have a name in PostgreSQL.
+		// It can either be an assigned name or an automatically generated one,
+		// if the constraint name is skipped in the query adding this constraint.
+		if len(pkName) > 0 { // ~ pkName != "" but without string comparison
```

Comment:
Let's just do `pkName != ""` if that's what we mean, or drop the comment? There's no reason to optimize this and make the code more obscure unless there's a benchmark showing it matters in a significant way. In this case I doubt it

Reply:
done in 08cc7bc476131bcd64f68c24c799c87ba0a21af0